sdk/communication/Azure.Communication.CallingServer/src/Models/CallAutomationModelFactory.cs

```diff
@@ -63,143 +64,80 @@ public static CreateCallResult CreateCallResult(CallConnection callConnection =
             return new CreateCallResult(callConnection, callConnectionProperties);
         }
 
+        /// <summary> Create an EventSource. </summary>
+        /// <param name="callConnectionId"> Call connection id for the event. </param>
+        /// <param name="eventName"> Optional event name; used for events related to content. </param>
+        /// <returns> A new <see cref="CallingServer.CreateCallResult"/> instance for mocking. </returns>
+        private static string CreateEventSource(string callConnectionId, string eventName = "")
```

Comment:
What is the purpose of this?

Reply:
resolve to unblock content. will fix in a later update.